OpenRA.Mods.Common/Traits/Player/PlaceBeacon.cs

```diff
@@ -14,12 +14,19 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	[Desc("A beacon that is constructed from a circle sprite that is animated once and a moving arrow sprite.")]
 	public class PlaceBeaconInfo : ITraitInfo
 	{
 		public readonly int Duration = 30 * 25;
 		public readonly string NotificationType = "Sounds";
 		public readonly string Notification = "Beacon";
-		public readonly string PalettePrefix = "player";
+
+		public readonly bool IsPlayerPalette = true;
+		[PaletteReference("IsPlayerPalette")] public readonly string Palette = "player";
+
+		public readonly string BeaconCollection = "beacon";
```

Comment:
The `Collection` terminology is used for UI artwork, and is incorrect for traits.  See the discussion in #10938.